samcli/commands/check/resources/LambdaFunction.py

```diff
@@ -0,0 +1,66 @@
+"""
+Class object for Lambda Functions. Contains object from template, as well as all data for lambda functions,
+excluding pricing info
+"""
+from samcli.commands.check.resources.TemplateResource import TemplateResource
+
+
+class LambdaFunction(TemplateResource):
+    def __init__(self, resource_object, resource_type):
```

Comment:
Do we need getters and setters here since everything is public?
Otherwise, directly referencing the members themselves is a lot cleaner.

Reply:
I will be making things private, so they will be necessary